R/logging.R

```diff
@@ -0,0 +1,110 @@
+#' Get node log files from Azure Storage. By default, this operation will print the files on screen.
+#'
+#' @param cluster The cluster object
+#' @param nodeId Id of the node
+#' @param type The type of logs the user wants: "stdout" or "stderr"
+#' @param verbose Flag for printing log files onto console
+#'
+#' @param ... Further named parameters
+#' \itemize{
+#'  \item{"localPath"}: { Path to save file to }
+#'  \item{"overwrite"}: { Will only overwrite existing localPath }
+#'}
+#'
+#' @examples
+#' stdoutText <- getClusterLogs(cluster, "tvm-1170471534_1-20170829t072146z", type = "stdout", verbose = FALSE)
+#' getClusterLogs(cluster, "tvm-1170471534_2-20170829t072146z", type = "stderr", localPath = "abc.txt", overwrite = TRUE)
```

Comment:
minor: line too long?

Reply:
Travis CI doesn't check for comments being too long